Controller/Payment/Checkout/Token.php

```diff
@@ -119,12 +119,13 @@ public function execute()
                 ->process();
             $token = $result->getToken();
             $transactionId = $result->getTransactionId();
+            $buyerId = $result->getBuyerId();
```

Comment:
Is this a magic method?

Reply:
Yes, we have set the data in Response/Transaction.php file